### ISSUE TITLE
Automated Tab Stops Focus Trap False Positive

### ISSUE THREAD

**Issue by sfoslund** [microsoft/accessibility-insights-web#5317]
## Describe the bug

A false positive focus trap failure is being reported through the new automated tab stops checks.

**To Reproduce**

1. Go to https://www.washington.edu/accesscomputing/AU/after.html
2. Click on the cheat sheet of a11y issues
3. Run tabstops over the modal dialog
4. Tab at least 3 times, so that focus loops back to the close ('X') button
5. See that focus trap failure is incorrectly reported on the close button

## Screenshots

[Screenshot of modal open with focus trap failure visualization over close button in top right corner]
![FocusTrapFalsePositive](https://user-images.githubusercontent.com/16010855/161346781-9d4484a5-2960-42c4-9548-a39b52d548f6.png)

## Additional context

Issue found when investigating https://github.com/microsoft/accessibility-insights-web/issues/5305

**Comment by ferBonnin** [comment]
reviewed with Peter

**Comment by sfoslund** [comment]
Resolved by #5705
